dtale/app.py

```diff
@@ -789,14 +783,15 @@ def get_instance(data_id):
     Returns a :class:`dtale.views.DtaleData` object for the data_id passed as input, will return None if the data_id
     does not exist
 
-    :param data_id: integer string identifier for a D-Tale process's data
-    :type data_id: str
+    :param data_id: integer identifier for a D-Tale process's data
+    :type data_id: int
     :return: :class:`dtale.views.DtaleData`
     """
-    data_id_str = global_state.find_data_id(str(data_id))
-    if data_id_str is not None:
+    if int(data_id) not in global_state.keys():
```

Comment:
If you wanted to take this one step further you could override the `__contains__` function on global_state so it can handle strings or integers.  Or just implement your own `contains` function to do it

Reply:
Good idea. I forgot about it existed.

Reply:
It didn't work. Even if __contains__ is exported, it still gives me error "module object is not iterable". So, there goes this solution and len(global_state) as well.